src/System.Diagnostics.FileVersionInfo/src/System.Diagnostics.FileVersionInfo.builds

```diff
@@ -9,6 +9,11 @@
       <OSGroup>Windows_NT</OSGroup>
     </Project>
     <Project Include="System.Diagnostics.FileVersionInfo.csproj">
+      <OSGroup>Windows_NT</OSGroup>
```

Comment:
Why is this a new configuration? I assume because there is no good way to share the Unix Any version with Windows_NT netcore50 version.